app/scripts/templates/permissions.mustache

```diff
@@ -1,20 +1,22 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="fxa-permissions-header">{{#t}}%(serviceName)s would like to know…{{/t}}</h1>
+    <h1 id="fxa-permissions-header">{{#t}}Request for permission{{/t}}</h1>
   </header>
 
   <section>
     <div class="error"></div>
     <div class="success"></div>
 
-    <p id="permissions">
-      <span class="permission">{{#t}}Email{{/t}}</span>
-      <span class="value">{{ email }}</span>
-    </p>
+    <p>{{#t}}%(serviceName)s wants to use the following info:{{/t}}</p>
+
+    <form id="permissions" novalidate>
+      <div class="input-row">
+        <label class="label-helper visible">{{#t}}Email{{/t}}</label>
```

Comment:
Hm, yeah. We should probably separate classes for behaviors and for styles, so we can easily share styles even when behavior differs.